File: test/LogTest.js
#!/usr/bin/env node
if ( require.main === module )
{
  var gepard = require ( "gepard" ) ;
  var c = gepard.getClient() ;
  c.log ( "SYSTEM LOG TEST JAVASCRIPT"
        , function()
          {
            this.setReconnect ( false ) ;
            this.end() ;
          }
        ) ;
}

File: python/xmp/LogTest.py
#!/usr/bin/env python

import os, sys, datetime, calendar

sys.path.insert ( 0, os.path.dirname(os.path.abspath(__file__) ) + "/../" )
from gepard import Client, util

client = Client.getInstance()

client.log ( "SYSTEM LOG TEST PYTHON" )
client.close()

date = datetime.datetime.now()
print ( util.formatDateAsRFC3339 ( date ) )
